src/Explorer/Menus/CommandBar/ConnectionStatusComponent.tsx

```diff
@@ -118,8 +122,7 @@ export const ConnectionStatus: React.FC<Props> = ({ container }: Props): JSX.Ele
     <>
       <TooltipHost
         content={
-          containerInfo?.status === ContainerStatusType.Active &&
-          Math.round(containerInfo.durationLeftInMinutes) <= Notebook.remainingTimeForAlert
```

Comment:
why remove this check

Reply:
On hover of tooltip by default showing time left in min